### DISCUSSION TITLE
Building profiles for device not owned

### DISCUSSION THREAD

**Discussion by salimv** [elgatosf/elgato#84] [Help] [answered]
Hi:

For the plugin we offer, we build and ship custom profiles for SD and XL devices along with the plugin. We would like to add a profile for mobile device. But I do not own a mobile subscription. Is it possible to create a mobile profile without needing a paid subscription for streamdeck mobile app ?

**Comment by ChekTek** [comment, marked as answer]
Hi @salimv 

Unfortunately, it is not possible to create profiles for devices you do not own at this time. You can reach out to `streamdeck.elgato@corsair.com` and let them know you are working on building a Stream Deck plugin. They should be able to gift you some subscription time to get your profiles prepared.
